Added bluGuide Addon

AniAniBobbani opened the pull request:
First upload of addon

AniAniBobbani committed:
Added bluGuide Annon

First upload of addon

AniAniBobbani committed:
copyright, tabs, settings

added copyright to all files, and fixed fields that were not filled out.

removed unneeded files in data folder.

Replaced some tabs with spaces

z16 commented on addons/bluguide/bluguide.lua:
```diff
@@ -0,0 +1,287 @@
+--Copyright (c) 2015, Anissa
```
Should be `©`

AniAniBobbani committed:
(c) and git files removed

nitrous24 commented on addons/bluguide/masterlist.lua:
```diff
@@ -0,0 +1,199 @@
+        flags = { draggable = false },
+        pos = {x = px, y = py} }
+    
+    slots = texts.new("Slots: 0/20 Points: 0/70", textsettings)
```
You set slots/points above but manually enter 20 and 70 here.

nitrous24 commented on addons/bluguide/masterlist.lua:
```diff
@@ -0,0 +1,199 @@
+    else
+        magicpoints = 0
+        spelllevel = 0
+        spellslots = 0
```
Here these 3 are called magicpoints spelllevel  and spellslots, but in the other 2  parts of the if statement they're just points, level and slots. Just wanna make sure this is intentional.

AniAniBobbani replied:
yes, lines 58-61 were totally unnecessary.  I'm removing them

AniAniBobbani committed:
Deleted unneeded/ old lines

old variables that had been renamed were set to 0 under certain
conditions.  the correct variables have these same values by default.

z16 commented on addons/bluguide/ui/spellboxes.lua:
```diff
@@ -0,0 +1,180 @@
+--LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+--ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+--(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+--SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
You have the copyright notice twice in here now ;)

z16 commented on addons/bluguide/res/traits.lua:
```diff
@@ -0,0 +1,427 @@
+--ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+--(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+--SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```
This is not really an issue, but just letting you know that the formatting in this file is all wonky now since you replaced tabs with spaces. They're interchangeable for indentation, but quite different for formatting. You may or may not wanna change that.

AniAniBobbani committed:
Removed duplicate copyright, changed formatting,

AniAniBobbani commented:
that formatting bothered me, so I just changed it all to single spaces.  Fixed the duplicate copyright and moved all copyrights to the bottom.
